website/docs/cloud-docs/api-docs/team-members.mdx

```diff
@@ -93,9 +93,58 @@ curl \
   https://app.terraform.io/api/v2/teams/257525/relationships/users
 ```
 
-## Delete a User from Team
+## Add a User to Team (With organization membership ID)
 
-This method removes multiple users from a team. Both users and teams must already exist. This DOES NOT delete the user; it only removes them from this team.
+This method adds multiple users to a team using the organization membership ID.
+
+`POST /teams/:team_id/relationships/organization-memberships`
+
+| Parameter  | Description         |
+| ---------- | ------------------- |
+| `:team_id` | The ID of the team. |
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+| Key path      | Type   | Default | Description                      |
+| ------------- | ------ | ------- | -------------------------------- |
+| `data[].type` | string |         | Must be `"organization-memberships"`.               |
+| `data[].id`   | string |         | The organization membership ID of the user to add. |
+
+### Sample Payload
+
+```json
+{
+  "data": [
+    {
+      "type": "organization-memberships",
+      "id": "ou-nX7inDHhmC3quYgy"
+    },
+    {
+      "type": "organization-memberships",
+      "id": "ou-tTJph1AQVK5ZmdND"
+    }
+  ]
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/teams/257525/relationships/organization-memberships
+```
+
+## Delete a User from Team (With user ID)
+
+This method removes multiple users from a team using the user ID. Both users and teams must already exist. This DOES NOT delete the user; it only removes them from this team.
```

Comment:
```suggestion
This method removes multiple users from a team using the user ID. Both users and teams must already exist. This method only removes a user from this team. It does not delete that user overall.
```

Suggest rewording to adding "method" after "This" to give "This" a subject. Change unneeded capitalization.